### PR TITLE
[V1] Release

### PR DESCRIPTION
**Pull request by rafaelsideguide**
I'm creating this draft for document all PRs we're building for the V1.

#506

### CODE REVIEW

**Comment by nickscamara** [comment]
- [ ] Convert Search into queue (v0), depr v1

**Review by Hridoy000368 — APPROVED** [review]

File: apps/api/requests.http
Comment (nickscamara) [inline apps/api/requests.http:1]: REVOKED

File: apps/api/src/controllers/v1/auth.ts
Comment (nickscamara) [inline apps/api/src/controllers/v1/auth.ts:1]: - [x] Add auth cache back

File: apps/api/src/controllers/v1/crawl-cancel.ts.WIP
Comment (nickscamara) [inline apps/api/src/controllers/v1/crawl-cancel.ts.WIP:1]: - [x] /crawl/cancel

File: apps/api/src/controllers/v1/crawl-status.ts
Comment (nickscamara) [inline apps/api/src/controllers/v1/crawl-status.ts:1]: we might wanna consider dropping auth here and just leave it on the uuid

File: apps/api/src/controllers/v1/crawl.ts
Comment (nickscamara) [inline apps/api/src/controllers/v1/crawl.ts:1]: need to match v0/crawl

File: apps/api/src/controllers/v1/map.ts
Comment (nickscamara) [inline apps/api/src/controllers/v1/map.ts:1]: - [x] Test log job

File: apps/api/src/controllers/v1/scrape.ts
Comment (nickscamara) [inline apps/api/src/controllers/v1/scrape.ts:1]: make sure it matches v0 exec

File: temp-37564.rdb
Comment (nickscamara) [inline temp-37564.rdb:1]: remove .rdb

**Review by nickscamara — COMMENTED** [review]
still some stuff but looking really good

**Comment by nickscamara** [comment]
- [ ] @rafaelsideguide check the above commit

File: apps/api/src/controllers/v1/crawl.ts
Comment (nickscamara) [reply apps/api/src/controllers/v1/crawl.ts:1]: Done

File: apps/api/src/controllers/v1/crawl-status.ts
Comment (nickscamara) [reply apps/api/src/controllers/v1/crawl-status.ts:1]: We will keep it w/ cache layer

File: apps/api/src/controllers/v1/scrape.ts
Comment (nickscamara) [reply apps/api/src/controllers/v1/scrape.ts:1]: yep

File: temp-37564.rdb
Comment (nickscamara) [reply temp-37564.rdb:1]: Done

File: apps/api/src/controllers/v1/crawl-cancel.ts.WIP
Comment (rafaelsideguide) [reply apps/api/src/controllers/v1/crawl-cancel.ts.WIP:1]: just implemented similar to v0 and added a e2e test

**Pull request merged**